3-projects/single-project-example.tf

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_folder" "single-project-folder" {
+  parent       = google_folder.app.id
+  display_name = "single-project-app"
+}
+
+module "single-project-app" {
```

Comment:
Perhaps we can have two examples here, one a lot more vanilla and one with all the networking goodies. (commented out, with link the readme)

Reply:
This sounds a great work around.

Reply:
Looks like most resources are `snake_case` so I'd update this module to also be `snake` instead of `kebap-case`

Reply:
yep agree snake_case would be good here

Reply:
nudge

Reply:
Done. Will come with next commit